src/mock/dialog.ts

```diff
@@ -157,4 +161,51 @@ const dialoger = [
     },
 ];
 
+
+export function opprettDialog(update:NyDialogMeldingData): DialogData & JSONObject {
+    const dialogId =
+        update.dialogId === undefined ? rndId() : `${update.dialogId}`;
+    const nyHenvendelse : HenvendelseData = {
+        id: rndId(),
+        dialogId: dialogId,
+        avsender: 'BRUKER',
+        avsenderId: "Z123456",
+        sendt: new Date().toISOString(),
+        lest: true,
+        tekst: update.tekst,
+    };
+
+    const eksisterendeDialoger = dialoger.filter(
```

Comment:
Tror denne kan skrives om til `const eksisterendeDialog = dialog.find((dialog) => dialog.id === dialogId);`

Reply:
jeg antar du mente dialoger.filter. Dialoger.find gir 'possibly undefined' for eksisterendeDialog, og feilmeldingen forsvinner ikke før find blir erstattet med filter (hjelper ikke med undefined sjekk)

Reply:
Gjorde en checkout av koden og å teste nå. 
Og det fungerer som forventet med `dialoger.find`, og en undefined-check

Reply:
Jeg ser det nå: endret slik da virker det: 
```
const eksisterendeDialog = dialoger.find(
        dialog => update.dialogId !== undefined && dialog.id === dialogId
    );
    if (eksisterendeDialog) {
```